xml/Accessibility/CAccPropServicesClass.xml

```diff
@@ -34,13 +34,13 @@
     <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5">
       <AttributeName>System.Runtime.InteropServices.TypeLibType(2)</AttributeName>
     </Attribute>
-    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netcore-3.0;netcore-3.1;net-5.0">
+    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netcore-3.0;netcore-3.1;net-5.0;dotnet-plat-ext-5.0">
```

Comment:
@mairaw Should there be `net-5.0` and `dotnet-plat-ext-5.0`?

Reply:
Yep, that concept still exists. .net-5.0 represents the shared framework.

Reply:
Now if it's correct for this API to exist in both I'm not sure

Reply:
And the API Catalog doesn't seem to have .NET 5.0 info yet
https://apisof.net/catalog/Accessibility.CAccPropServicesClass

/cc @safern @terrajobst

Reply:
Yeah, working on that as well. https://github.com/dotnet/api-catalog-infra

Should have it next week.